CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ The sections should follow the order `Added`, `Changed`, `Fixed`, and `Removed`.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Added
+
+- On macOS, read `AppleFontSmoothing` from user defaults to decide whether fonts should be "smoothed".
+
+### Changed
+
+- `darwin::Rasterizer` renamed to `darwin::CoreTextRasterizer`
+
+### Fixed
+
+- Get thin strokes working on macOS Big Sur and Monterey, enabled with `AppleFontSmoothing` set to 0.
```

Comment:
```suggestion
- `AppleFontSmoothing` not being respected when smoothing fonts on macOS
```

Reply:
I was struggling with how to phrase this, but I feel like this is redundant with what is in "Added". The thing which is fixed is that there's no way to use thin strokes currently, and now there will be. I'll try a different wording.

Reply:
We could remove the "Fixed" part altogether. I "fixed" the broken thing by removing it and adding something else. What do you think?

Reply:
Updated.